cluster-scope/overlays/ocp-staging/groups/local-cluster-admins.yaml

```diff
@@ -6,4 +6,3 @@ metadata:
     kustomize.config.k8s.io/behavior: replace
 users:
   - mhaley@redhat.com
```

Comment:
@larsks is `mhaley@redhat.com` part of the project?

Reply:
No, that was Margaret (Red Hat co-op), who is no longer around.

Reply:
Updated the PR to remove the entire file.